.gitignore

```diff
@@ -9,6 +9,7 @@
 *.swp
 .vimrc
 .kubeconfig
+.openshiftconfig
```

Comment:
I retract my comment... `./.openshiftconfig` is not a known filename we generate